### ISSUE TITLE
global_pose.csv to local_pose

### ISSUE THREAD

**Issue by cristianrubioa** [irapkaist/file_player_mulran#6]
Hi, how could i convert global poses to local map i.e something like this.
![Screen Shot 2021-02-18 at 2 06 28 AM](https://user-images.githubusercontent.com/44789319/108318491-ec141a80-718d-11eb-85c5-9cadaf9520b7.png)

regarding this https://github.com/irapkaist/file_player_mulran/issues/5  I get a map of (latitude, longitude), but how could I get a result like the previous one from this map?
![Screen Shot 2021-02-18 at 2 10 20 AM](https://user-images.githubusercontent.com/44789319/108318808-765c7e80-718e-11eb-9bca-1ec4ac1d75f3.png)

**Comment by VRER1997** [comment]
I have the same problem. Do you have a solution, please?

**Comment by 0ontheroad** [comment]
> I have the same problem. Do you have a solution, please?

I haven't encountered your problem yet. How did you get the first picture above? I reported the following error with evo（global_pose.csv）:
`[ERROR] EuRoC MAV state ground truth must have 17 entries per row and no trailing delimiter at the end of the rows (comma)`

**Comment by gisbi-kim** [comment]
@cristianrubioa, @VRER1997 

see here please, https://github.com/gisbi-kim/mulran-evaluation

tldr: the global coord GT (in earth coord) is converted into ouster-oriented (starting point is identity w.r.t the ouster)
so you can compare some lidar odometry (e.g., https://github.com/gisbi-kim/SC-LIO-SAM#applications) whose start is always identity with respect to the converted GT trajectory.

**Comment by cristianrubioa** [comment]
> > I have the same problem. Do you have a solution, please?
> 
> I haven't encountered your problem yet. How did you get the first picture above? I reported the following error with evo（global_pose.csv）:
> `[ERROR] EuRoC MAV state ground truth must have 17 entries per row and no trailing delimiter at the end of the rows (comma)`

Hi @0ontheroad 
The image above was obtained by extracting the trajectory of the [A-LOAM](https://github.com/HKUST-Aerial-Robotics/A-LOAM) method.
I extracted the trajectory in TUM format. In this [repo](https://github.com/cristianrubioa/extract_trajectory) you can review about it.  (The explanation is intuitive, but you can [ask me](https://github.com/cristianrubioa/extract_trajectory/issues) there in any case).

**Comment by cristianrubioa** [comment]
@gisbi-kim 
Thanks !
I recently tried it and everything is fine.
